tensorboard/uploader/uploader_main.py

```diff
@@ -348,23 +358,16 @@ def execute(self, server_info, channel):
         )
         gen = exporter_lib.list_experiments(api_client, fieldmask=fieldmask)
         count = 0
+
+        if self.json:
+            formatter = formatters.JsonFormatterI()
```

Comment:
Presumably `JsonFormatterI` should be `JsonFormatter`?

This breaks when run:

```
    formatter = formatters.JsonFormatterI()
AttributeError: module 'tensorboard.uploader.formatters' has no attribute 'JsonFormatterI'
```

Reply:
Hmmm. I don't know how this got in. I'll be more careful with my keyboard setup. Thanks for catching this.